src/controller/CHIPDevice.h

```diff
@@ -372,6 +361,13 @@ class DLL_EXPORT Device
      */
     CHIP_ERROR LoadSecureSessionParametersIfNeeded(bool & didLoad);
 
+    /**
+     * @brief
+     *   Initialize internal command sender, required for sending commands over interaction model.
```

Comment:
Remove the extra space before 'Initialize'

Reply:
Done